sandbox/src/server.ts

```diff
@@ -66,30 +69,51 @@ export async function returnSchema(): Promise<GraphQLSchema> {
 }
 
 async function cohortApiMockserver() {
-  const server = new ApolloServer<MockserverContext>({
-    schema: await returnSchema(),
+    const app = express();
+    // const httpServer = http.createServer(app);
+    const server = new ApolloServer<MockserverContext>({
+        schema: await returnSchema(),
 
-    plugins: [requestLogger, formatResponse],
-  });
+        plugins: [requestLogger, formatResponse/*, ApolloServerPluginDrainHttpServer({httpServer})*/],
+    });
 
   async function start() {
-    const { url } = await startStandaloneServer(server, {
-      context: async () => ({
-        allCohorts: data
-      }),
-      listen: { port },
-    });
-    console.log(`Cohort api mockserver running at ${url}`);
+    await server.start()
+
+    app.use(
+        '/',
+        cors<cors.CorsRequest>(),
+        bodyParser.json({ limit: '50mb' }),
```

Comment:
See https://github.com/NHSDigital/caas-published-cohort-definitions/pull/18#discussion_r1360463593 for a discussion on body size limits. This needs updating post-rebase.